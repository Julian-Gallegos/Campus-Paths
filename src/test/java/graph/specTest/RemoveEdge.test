# Tests that an edge between a parent node and a child node can be removed.

# Create an empty graph
CreateEmptyGraph graph1

# Add a pair of nodes
AddNode graph1 node1
AddNode graph1 node2

# Add an edge from node1 to node 2
AddEdge graph1 node1 node2 edge1

# Remove an edge from node1 to node 2
RemoveEdge graph1 node1 node2 edge1