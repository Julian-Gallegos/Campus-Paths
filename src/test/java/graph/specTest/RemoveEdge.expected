# Tests that an edge between a parent node and a child node can be removed.

#Create an empty graph
created graph graph1

# Add node to a graph
added node node1 to graph1
added node node2 to graph1

# Add an edge from node1 to node 2
added edge edge1 from node1 to node2 in graph1

# Remove an edge from node1 to node 2
removed edge edge1 from node1 to node2 in graph1